db/seeds.rb

```diff
@@ -135,15 +137,18 @@
 
 p 'assigning worksheets to work groups'
 
-work_groups.each_with_index do |work_group, idx|
-  template = idx == WorkGroup.count - 1 ? WorksheetTemplate.last : WorksheetTemplate.first
-  Worksheet.create!(
-    title: "Worksheet #{idx}",
-    canvas: '',
-    worksheet_template: template,
-    work_group: work_group,
-    template_image_url: 'https://res.cloudinary.com/naokimi/image/upload/v1563422680/p7ojmgdtwshkrhxmjzh1.jpg'
-  )
+work_groups = WorkGroup.all
+Classroom.all.each_with_index do |classroom, index|
+  template = index % 2 == 0 ? WorksheetTemplate.last : WorksheetTemplate.first
+  classroom.work_groups.each_with_index do |work_group, index|
```

Comment:
rename the `index` here to `idx` as it's confusing inside this loop which index you are referring too